api/s2n.h

```diff
@@ -152,7 +160,9 @@ extern int s2n_config_disable_x509_verification(struct s2n_config *config);
 extern int s2n_config_set_max_cert_chain_depth(struct s2n_config *config, uint16_t max_depth);
 
 extern int s2n_config_add_dhparams(struct s2n_config *config, const char *dhparams_pem);
+S2N_DEPRECATED
 extern int s2n_config_set_cipher_preferences(struct s2n_config *config, const char *version);
```

Comment:
These might be more clear if they're on the same line as the deprecated API. Like:
```
S2N_DEPRECATED extern int s2n_config_set_cipher_preferences(struct s2n_config *config, const char *version);
```

Reply:
`S2N_API` proceeds all other definitions, so this is now matching.